cmyt 2.0.2

msentissi opened the pull request:
cmyt 2.0.2 

**Destination channel:** Defaults

### Links

- [PKG-7098]
- dev_url:        https://github.com/yt-project/cmyt//tree/v2.0.2
- conda_forge:    https://github.com/conda-forge/cmyt-feedstock
- pypi:           https://pypi.org/project/cmyt/2.0.2
- pypi inspector: https://inspector.pypi.io/project/cmyt/2.0.2

### Explanation of changes:

- new version number


[PKG-7098]: https://anaconda.atlassian.net/browse/PKG-7098?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

msentissi committed:
upgrade to v2.0.2

ELundby45 reviewed (approved):
Looks like tests aren't included from pypi. They are in the github release tarball if we wanted to add those back.

danpetry commented on recipe/meta.yaml:
```diff
@@ -1,48 +1,41 @@
     - python
-    - setuptools
+    - hatchling
     - wheel
```
you don't need wheel if using hatchling, it implements its own wheel build stuff

msentissi replied:
I left it in because the linter complains if `wheel` is missing for a `pypi` package. Is it preferable to ignore it ?

danpetry replied:
probably, yes. even better, add the fix to the linter

danpetry replied:
as an aside, I wouldn't let that hold up this PR, so recommend ignoring the lint check and changing the rule in the linter as well

msentissi replied:
It's actually no longer an issue for this PR since I've switched to GH as a source.
EDIT: It also seems that the linter already has [exceptions in place to the missing_wheel error](https://github.com/anaconda/anaconda-linter/blob/04ea432c25b1276b7425c284923b635f68b3241f/anaconda_linter/lint/check_build_help.py#L381) when certain build backends are used.

msentissi committed:
add tests

msentissi commented:
> Looks like tests aren't included from pypi. They are in the github release tarball if we wanted to add those back.

Switched to the github release and added tests back in [b38c733](https://github.com/AnacondaRecipes/cmyt-feedstock/pull/3/commits/b38c733de36d9bb59e144c8ae5d4866f7922fcc6).

msentissi committed:
add staging channel

msentissi committed:
remove staging channels
